src/coreclr/debug/daccess/request.cpp

```diff
@@ -1232,9 +1232,9 @@ HRESULT ClrDataAccess::GetTieredVersions(
             PTR_Module pModule = (PTR_Module)pMD->GetModule();
             if (pModule->IsReadyToRun())
             {
-                PTR_PEImageLayout pImage = pModule->GetReadyToRunInfo()->GetImage();
+                PTR_ReadyToRunLoadedImage pImage = pModule->GetReadyToRunInfo()->GetImage();
                 r2rImageBase = dac_cast<TADDR>(pImage->GetBase());
-                r2rImageEnd = r2rImageBase + pImage->GetSize();
+                r2rImageEnd = r2rImageBase + pImage->GetVirtualSize();
```

Comment:
This has worked until now because the `.text` section has always been the first one in the PE and had the same physical offset and virtual address (so definitely always ended before the physical size of the file), but this should be virtual size for correctness anyway.